### PR TITLE
Added Mohit Sharma as a Mentor

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [X] Included a Preview link and screenshot showning after and before the changes.
- [X] Included a description of change below.
- [X] Squashed the commits.

# Changes done in this Pull Request
Added Mohit Sharma as a mentor
https://arjunpandey2003.github.io/gci18.arjunpfossasia.orgt/
## Description / Changes

### CODE REVIEW

File: _data/mentors.yml [hunk 1]
```diff
@@ -36,3 +36,7 @@
 - name: Sanskar Jethi
   image: sanskar.jpg
   github: stealthanthrax
+
+- name: Mohit Sharma
+  image: ms.jpeg
```
Comment (simsausaurabh) [inline _data/mentors.yml]: Try to keep the image format consistent with other images (Use jpg).

Comment (arjunpandey2003) [reply _data/mentors.yml]: ok

Comment (arjunpandey2003) [reply _data/mentors.yml]: doing

Comment (simsausaurabh) [reply _data/mentors.yml]: Squash your commit into one at the end.

Comment (arjunpandey2003) [reply _data/mentors.yml]: hey some problem in my git desktop, resolving closing pr, making new one, sorry for the confusion

**Pull request closed without merging**